articles/connections/database/password-change.md

```diff
@@ -12,36 +12,39 @@ useCase: customize-connections
-
-We strongly encourage you to enable **Change Password flow v2**. To learn more about the vulnerability and migration, please see [Vulnerable Password Flow](/migrations/past-migrations#vulnerable-password-flow). To learn more about migrating to Lock 11, please take a look at the [Lock 11 Migration Guide](/libraries/lock/v11/migration-guide).
+This article will help you learn how to reset a user's password. If you are trying to configure the custom Password Reset page for your tenant, see [Password Reset Page](/universal-login/password-reset). If you are a user of Auth0 and are trying to reset the password to your Auth0 account, see [Reset Your Auth0 Account Password](/support/reset-account-password).
 :::
 
 There are two basic methods of changing a password:
-- an [interactive password reset flow](#interactive-password-reset-flow) where the user receives an email with a link that opens an Auth0 hosted page to enter the new password.
+
+- an [interactive password reset flow](#interactive-password-reset-flow) where the user receives an email with a link that opens the Auth0 password reset page to enter the new password.
 - [directly setting the new password](#directly-setting-the-new-password) either using the Management API v2 or the Dashboard.
 
 ::: note
-You can only change passwords for users signing in using database connections. Users signing in using social or enterprise connections need to reset their passwords with the appropriate system.
+You can only change passwords for users signing in using database connections. Users signing in using social or enterprise connections need to reset their passwords with the relevant identity provider.
 :::
 
 ## Trigger an interactive password reset flow
 
-An interactive password reset flow can be triggered in two ways:
+An interactive password reset flow can be triggered in three ways, depending on your use case:
+
+- [**Universal Login Page**](#use-universal-login): If your app uses Universal Login, the user uses the Lock widget on the Login screen to trigger a password reset email.
+- [**Authentication API**](#use-the-authentication-api): Send a `POST` call to the Authentication API to send a password reset email to the user.
+- [**Embedded Lock**](#use-lock): The user uses the Lock login screen embedded in their app to trigger a password reset email.
+
+### Use Universal Login
 
-+ [**Authentication API**](#using-the-authentication-api): Send a `POST` call to the Authentication API to send a password reset email to the user.
-+ [**Lock**](#using-lock): The user uses the Lock login screen to trigger a password reset email.
+If your application is using Universal Login, the user will be able to trigger a password reset from the login page. With the New Universal Login Experience, the user will click the **Don't remember your password?** link and then enter their email address.
+
+::: note
```

Comment:
The experience is basically the same for the new and old UL, do we need to talk about both?